mutexbot/action.yml

```diff
@@ -0,0 +1,23 @@
+# This file is generated by ghactions
+# Do not edit this file manually unless you disable the `generate` feature.
+
+name: Mutexbot
+description: Action to reserve and release resources on mutexbot.com
+inputs:
+  api_key:
+    description: Mutexbot API key
+  mode:
+    description: Whether to reserve or release or force-release
+    default: reserve
+  resource_name:
+    description: Mutexbot resource name
+  duration:
+    description: Duration for which to reserve
+    default: ''
+  isolation_channel:
+    description: Isolation channel of resource
+    default: ''
+outputs: {}
+runs:
+  using: docker
+  image: docker://neondatabase/dev-actions:mutexbot-pr-7
```

Comment:
I guess we'll need to change it?

Reply:
Yes. Due to this we have to start using version tags for the action now, but that shouldn't be a problem I think.